src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -328,17 +340,40 @@ public static ConfigDef getConfig() {
     everything.putAll(StorageCommonConfig.getConfig().configKeys());
     everything.putAll(PartitionerConfig.getConfig().configKeys());
 
-    Set<String> blacklist = new HashSet<>();
-    blacklist.add(StorageSinkConnectorConfig.ROTATE_INTERVAL_MS_CONFIG);
-    blacklist.add(StorageSinkConnectorConfig.ROTATE_SCHEDULE_INTERVAL_MS_CONFIG);
-    blacklist.add(StorageSinkConnectorConfig.SHUTDOWN_TIMEOUT_CONFIG);
+    Set<String> skip = new HashSet<>();
+    skip.add(StorageSinkConnectorConfig.ROTATE_SCHEDULE_INTERVAL_MS_CONFIG);
+    skip.add(StorageSinkConnectorConfig.SHUTDOWN_TIMEOUT_CONFIG);
 
     ConfigDef visible = new ConfigDef();
+
+    visible.define(
+        STORAGE_CLASS_CONFIG,
+        Type.CLASS,
+        Importance.HIGH,
+        STORAGE_CLASS_DOC,
+        "Storage",
+        1,
+        Width.NONE,
+        STORAGE_CLASS_DISPLAY
```

Comment:
Should these define the default value? If so, then `addDefaults` shouldn't be needed, right?